extensions/interactions/CodeRepl/CodeReplPredictionService.js

```diff
@@ -298,8 +298,11 @@ oppia.factory('CodeReplPredictionService', [
         var programVector = CountVectorizerService.vectorize(
           tokenizedProgram, cvVocabulary);
 
-        prediction = SVMPredictionService.predict(svmData, programVector);
-        return prediction;
+        predictionResult = SVMPredictionService.predict(svmData, programVector);
+        if (predictionResult.predictionConfidence > 0.5) {
```

Comment:
Perhaps explain where 0.5 comes from and under which circumstances we might want to modify it; consider making it into a named constant.